gqueries/modules/energy_mix/supply_chart/factsheet_supply_fossil_other.gql

```diff
@@ -4,7 +4,6 @@
         V(energy_distribution_coal, demand) - V(energy_export_coal_baseload, demand) - V(energy_export_coal_backup,demand),
         V(energy_distribution_lignite, demand) - V(energy_export_lignite_baseload, demand),- V(energy_export_lignite_backup, demand),
         V(energy_distribution_uranium_oxide, demand) - V(energy_export_uranium_oxide, demand),
-        V(energy_distribution_non_biogenic_waste, demand) - V(energy_export_non_biogenic_waste, demand),
```

Comment:
Shouldn't `V(energy_distribution_non_biogenic_waste, demand)` be kept in the query?